ports/samd/boards/ADAFRUIT_QTPY_SAMD21/board.json

```diff
@@ -0,0 +1,20 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "USB-C"
+    ],
+    "images": [
+        "qt_py_samd21.jpg"
+    ],
+    "mcu": "samd21",
+    "product": "QT PY (SAMD21)",
```

Comment:
Looks like the name is "QT Py - SAMD21".

Reply:
Done